package.json

```diff
@@ -49,72 +47,70 @@
-    "start":                  "npm run build:www && ionic serve --no-livereload --nogulp -s --address 0.0.0.0",
-    "start:ios":              "npm run build:www && npm run build:ios && npm run open:ios",
-    "start:android":          "npm run build:www && npm run build:android && npm run run:android",
-    "start:windows":          "npm run build:www && npm run build:windows",
-    "start:desktop":          "npm start",
-    "watch":                  "grunt watch",
-    "build:www":              "grunt",
-    "build:www-release":      "grunt prod",
-    "build:ios":              "cordova prepare ios && cordova build ios --debug",
-    "build:android":          "cordova prepare android && cordova build android --debug",
-    "build:windows":          "cordova prepare windows && cordova build windows -- --arch=\"ARM\"",
-    "build:ios-release":      "cordova prepare ios && cordova build ios --release",
-    "build:android-release":  "cordova prepare android && cordova build android --release",
-    "build:windows-release":  "cordova prepare windows && cordova build windows --release --arch=\"ARM\"",
-    "build:desktop":          "grunt desktop",
-    "build:osx":            "grunt osx",
-    "open:ios":               "open platforms/ios/*.xcodeproj",
-    "open:android":           "open -a open -a /Applications/Android\\ Studio.app platforms/android",
-    "final:www":              "npm run build:www-release",
-    "final:ios":              "npm run final:www && npm run build:ios-release && npm run open:ios",
-    "final:android":          "npm run final:www && npm run build:android-release && npm run sign:android && npm run run:android-release",
-    "final:windows":          "npm run final:www && npm run build:windows-release",
-    "final:desktop":          "npm run build:desktop && npm run build:osx",
-    "run:android":            "cordova run android --device",
-    "run:android-release":            "cordova run android --device --release",
-    "log:android":            "adb logcat | grep chromium",
-    "sign:android":           "rm -f platforms/android/build/outputs/apk/android-release-signed-aligned.apk; jarsigner -verbose -sigalg SHA1withRSA -digestalg SHA1 -keystore ../copay.keystore -signedjar platforms/android/build/outputs/apk/android-release-signed.apk  platforms/android/build/outputs/apk/android-release-unsigned.apk copay_play && $ANDROID_HOME/build-tools/26.0.1/zipalign -v 4 platforms/android/build/outputs/apk/android-release-signed.apk platforms/android/build/outputs/apk/android-release-signed-aligned.apk",
-    "test":                   "echo \"no package tests configured\"",
-    "clean":                  "trash platforms && trash plugins && cordova prepare",
-    "unstage-package":        "git reset package.json",
-    "clean-all":              "git clean -dfx"
+    "postinstall": "sed -i  -e \"s/.*require(...\\/...);//g\" node_modules/asn1.js-rfc5280/index.js; bower install",
```

Comment:
I believe there are better npm implementations for scripts that we can utilize to clean this up. Low priority.